runner_manager/logging.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from runner_manager.models.settings import Settings
+
+settings = Settings()
+
+log_level = getattr(logging, settings.log_level.upper(), logging.INFO)
+log = logging.getLogger(settings.name)
+log.setLevel(log_level)
+
+# console handler
+console_handler = logging.StreamHandler()
```

Comment:
What does it do?

Reply:
it adds the handler to log the console stdout